config.example.wikimedia.yaml

```diff
@@ -24,6 +24,8 @@ default_project: &default_project
             host: http://graphoid.wikimedia.org
           mathoid:
             host: http://mathoid-tester.wmflabs.org
+            # 10 days Varnish caching, one day client-side
+            cache-control: s-maxage=864000, max-age=86400
```

Comment:
`config.test.yaml` should have the same change. Also, it'd be nice to add this header's check in the `x-amples` section of `v1/mathoid.yaml`